dot-net-sdk/eppo-sdk.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageProjectUrl>https://github.com/Eppo-exp/dot-net-server-sdk</PackageProjectUrl>
     <RepositoryUrl>https://github.com/Eppo-exp/dot-net-server-sdk</RepositoryUrl>
     <PackageId>Eppo.Sdk</PackageId>
-    <Version>3.1.1</Version>
+    <Version>3.2.1</Version>

```

Comment:
This got out of sync with the repo tag :(
the last release, v3.2.0 was a minor change, this release represents a bugfix to 3.2.0, thus we are releasing `3.2.1` back in sync with the repo tag and named release.